packages/grafana-data/src/valueFormats/categories.ts

```diff
@@ -183,8 +183,8 @@ export const getCategories = (): ValueFormatCategory[] => [
       { name: 'tebibits/sec', id: 'Tibits', fn: binaryPrefix('b/s', 4) },
       { name: 'terabytes/sec', id: 'TBs', fn: SIPrefix('B/s', 4) },
       { name: 'terabits/sec', id: 'Tbits', fn: SIPrefix('b/s', 4) },
-      { name: 'petibytes/sec', id: 'PiBs', fn: binaryPrefix('B/s', 5) },
-      { name: 'petibits/sec', id: 'Pibits', fn: binaryPrefix('b/s', 5) },
+      { name: 'pebibytes/sec', id: 'PiBs', fn: binaryPrefix('B/s', 5) },
+      { name: 'pebibits/sec', id: 'Pibits', fn: binaryPrefix('b/s', 5) },
```

Comment:
interesting -- I have never heard of pebibytes!
https://www.techtarget.com/searchstorage/definition/pebibyte-PiB